sledgehammer/start-up.sh

```diff
@@ -149,6 +149,11 @@ echo "# Sledgehammer added to log to the admin node" >> /etc/rsyslog.conf
 echo "*.* @@${ADMIN_IP}" >> /etc/rsyslog.conf
 service $RSYSLOGSERVICE restart
 
+# Sometimes at this point network is not up yet, wait for it
+ping -c 1 -w 60 $ADMIN_IP > /dev/null || {
```

Comment:
I don't think you want `ping -c 1 -w 60`; I think you want to repeat 60 times `ping -c 1 -w 1`. It's slightly different as in the current code, the echo request is only sent once, and might be lost.